src/class/audio/audio.h

```diff
@@ -663,15 +663,16 @@ typedef struct TU_ATTR_PACKED {
 } audio_desc_channel_cluster_t;
 
 /// AUDIO Class-Specific AC Interface Header Descriptor (4.7.2)
+// https://www.usb.org/sites/default/files/midi10.pdf (B.3.2) ...tested against a real device
 typedef struct TU_ATTR_PACKED
 {
-  uint8_t bLength            ; ///< Size of this descriptor in bytes: 9.
-  uint8_t bDescriptorType    ; ///< Descriptor Type. Value: TUSB_DESC_CS_INTERFACE.
-  uint8_t bDescriptorSubType ; ///< Descriptor SubType. Value: AUDIO_CS_AC_INTERFACE_HEADER.
-  uint16_t bcdADC            ; ///< Audio Device Class Specification Release Number in Binary-Coded Decimal. Value: U16_TO_U8S_LE(0x0200).
-  uint8_t bCategory          ; ///< Constant, indicating the primary use of this audio function, as intended by the manufacturer. See: audio_function_t.
```

Comment:
This changes definition from Audio 2 specification section 4.7.2 to version 1.0 section 4.3.2 with limitation to 1 interface.

Maybe better way to handle this is to have 1.0 version separated and not change just one structure to different specification version.

Reply:
Ah. Yes. I didn't realise that was the root of the problem! 

I _would_ suggest the v1 be called `audio_desc_cs_ac_interface_t` and v2 `audio_desc_cs_ac_interface_v2_t` ...but I guess that might be seen as an undesirable breaking-change :/

Do you have any ideas for the best/right way to do this?